crates/hyperqueue/src/dashboard/data/mod.rs

```diff
@@ -44,6 +45,30 @@ impl DashboardData {
     pub fn worker_count_at(&self, time: SystemTime) -> usize {
         self.worker_count_timeline.get_worker_count(time)
     }
+
+    pub fn get_latest_overview(&self) -> Vec<WorkerOverview> {
```

Comment:
Maybe it will have to be changed later, but for now we can simply return references to avoid clones. `-> Vec<&WorkerOverview>`. Same in `get_last_overview_for`.